src/components/ListItem.jsx

```diff
@@ -51,6 +59,7 @@ export function ListItem({ name, itemId, dateLastPurchased }) {
 					/>
 					{name}
 				</label>
+				<button onClick={handleDelete}>Remove</button>
```

Comment:
A button element's default `type` is submit. Since you're not submitting a form here, I suggest to specify the type to `<button type="button">` !

Reply:
Thanks for the tip!